holoviews/operation/element.py

```diff
@@ -700,6 +700,14 @@ class histogram(Operation):
       Used for setting a common style for histograms in a HoloMap or AdjointLayout.""")
 
     def _process(self, element, key=None):
+
+        bad_bins_msg = 'bins only accept numerical data.'
```

Comment:
I think this message could be improved a bit. How about:

'Histogram binning can only be performed on numerical data'